tmt.spec

```
@@ -64,7 +64,7 @@ Obsoletes:      tmt-provision-container < %{version}-%{release}
 %endif
 Requires:       tmt == %{version}-%{release}
 Requires:       podman
-Requires:       (ansible or ansible-collection-containers-podman)
+Requires:       tmt-ansible == %{version}-%{release}
```

```suggestion
Requires:       tmt+ansible == %{version}-%{release}
```

The plus sign should be used in the `Requires` sections as well, right?